src/browser/modules/D3Visualization/lib/visualization/components/layout.ts

```diff
@@ -71,8 +71,11 @@ const layout: AvailableLayouts = {
         )
 
       const simulation = forceSimulation<VizNode, Relationship>()
-        .force('charge', forceManyBody().strength(-300))
-        .on('tick', () => render())
+        .force('charge', forceManyBody().strength(-400))
+        .on('tick', () => {
+          simulation.tick(10)
```

Comment:
Did you consider doing a variable amount of ticks here? I wonder if ticking as many times as possible within 1/60th of a second (or 1/100th of a second if we want to leave some time for the rest of the code to run) would make it even better?

Reply:
Nice! Good to know